.circleci/config.yml

```diff
@@ -0,0 +1,171 @@
+# Javascript Node CircleCI 2.0 configuration file
```

Comment:
Could we move to github actions, since github actions is already integrated, I haven't used circleci for a while, plus github actions is free for OOP, what do you think?

Reply:
That sounds good, but I have no knowledge of how to use it.

Not really into learning it right now, due to the limited time available.

If you are able to rewrite it into Github Action it would be lit 👌.

Reply:
true

Reply:
https://app.circleci.com/pipelines/github/stoqey/ib?invite=true

Reply:
+ @Tsopic